live/src/main/java/org/dbpedia/extraction/live/feeder/AllPagesFeeder.java

```diff
@@ -0,0 +1,99 @@
+package org.dbpedia.extraction.live.feeder;
+
+import org.dbpedia.extraction.live.core.LiveOptions;
+import org.dbpedia.extraction.live.queue.LiveQueueItem;
+import org.dbpedia.extraction.live.queue.LiveQueuePriority;
+import org.dbpedia.extraction.live.util.DateUtil;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+
+/**
+ * @author Lukas Faber, Stephan Haarmann, Sebastian Serth
+ *         date 02.07.2016.
+ */
+public class AllPagesFeeder extends Feeder {
+
+    private boolean isFinished = false;
+    private String query_base =
+            "?action=query&format=json&list=allpages&aplimit=500&apnamespace=%d&continue=%s&apcontinue=%s";
```

Comment:
In general, scala has very good support for XML with type safety, I suggest we switch to XML here and be consistent with the rest of the Api calls